gtests/test_nopython_transform.cpp

```diff
@@ -7,11 +7,22 @@
 #error "Python.h should not be included."
 #endif
 
-template <typename T>
+template <typename T, bool IsPow2>
+struct FFTTestParams
+{
+    using Type = T;
+    static constexpr bool isPow2 = IsPow2;
+};
```

Comment:
Typed test parameter template declaration.
The first template argument specifies the floating point type (`double` or `float`), while the second indicates whether the input data length is a power of 2 (radix-2) or not.